nuxeo-drive-client/tests/test_remote_deletion.py

```diff
@@ -34,44 +36,40 @@ def test_synchronize_remote_deletion(self):
         # Get local and remote clients
         local = self.local_client_1
         remote = self.remote_document_client_1
+        remote_admin = self.root_remote_client
+        Options.server_version = self.engine_1.get_server_version()
```

Comment:
It is not necessary as it will be set via https://github.com/nuxeo/nuxeo-drive/pull/434/files#diff-81afa6dfa306390281c59e4e8adbe80eR121. Right?